pkg/install/generate_test.go

```diff
@@ -72,11 +73,22 @@ func TestInstallerPodSpec(t *testing.T) {
 			},
 			provisionName:  "testprovision",
 			skipGatherLogs: true,
+			extraEnvVars: []corev1.EnvVar{
+				{
+					Name:  "TESTVAR",
+					Value: "TESTVAL",
+				},
+			},
 			validate: func(t *testing.T, actualPodSpec *corev1.PodSpec, actualError error) {
 				expectedPodMemoryRequest := resource.MustParse("800Mi")
 				actualPodMemoryRequest := actualPodSpec.Containers[2].Resources.Requests[corev1.ResourceMemory]
 
 				assert.Equal(t, expectedPodMemoryRequest, actualPodMemoryRequest, "Incorrect pod memory request")
+
+				for _, container := range actualPodSpec.Containers {
+					assert.Equal(t, "TESTVAR", container.Env[1].Name, "Environment variable name not passed to pod spec")
```

Comment:
Why the hardcoded `1` index. Is something like `assert.Contains(t, container.Env, corev1.EnvVar{Name / Value})` usable?